flatlaf-core/src/main/java/com/formdev/flatlaf/ui/FlatListUI.java

```diff
@@ -318,6 +323,18 @@ protected void paintCell( Graphics g, int row, Rectangle rowBounds, ListCellRend
 			cx = rowBounds.x;
 			cw = rowBounds.width;
 		}
+		
+		// filelist/combobox does not support alternate row color
+		if( !isFileList && !(dataModel instanceof DefaultComboBoxModel) ) {
```

Comment:
Better use `"ComboBox.list".equals( list.getName() )` to check whether list is used in combobox. Not all comboboxes use `DefaultComboBoxModel`. E.g. "Look in" combobox in `JFileChooser`.

Why exclude filelist? Seems to work ok.

Reply:
Just fixed the check and reworked the filelist. 

Thanks

--